client/src/api/common/data.ts

```diff
@@ -220,3 +220,17 @@ export function startBoost() {
 export function stopBoost() {
   emit('stop-boost');
 }
```

Comment:
We can probably remove `startLogging()`, `stopLogging()`, `startBoost()` and `stopBoost()`, but just double check we don't use it anywhere